projects/lib-shared-modules/src/lib/components/dialogs/dialog-popup/dialog-popup.component.html

```diff
@@ -116,9 +116,9 @@ <h3 class="text-sm font-bold p-4" >{{dialogueData?.content |translate}}</h3>
       <div  class=" w-[100%] sm:w-[80%]" #certificateContainer>
       </div>
     </div>
-
+    
 </div>
-<div class="border-t"></div>
+<div [ngStyle]="{'border-top':  dialogRef._containerInstance._config.height === '75%' ? 'none' : '1px solid #ccc'}"></div>
```

Comment:
if  button is there then show border-t  else not show border- t    , instead of checking height and adding class